taui/package.json

```diff
@@ -50,6 +50,8 @@
     "jsolines": "~2.1.0",
     "leaflet": "~1.3.4",
     "lodash": "~4.17.11",
+    "logrocket": "^0.6.19",
```

Comment:
Nitpick: let's pin the version numbers here with `~` and specify `3.0.0` for the react package.